events/Dockerfile.receiver

```diff
@@ -0,0 +1,24 @@
+# Build
+FROM golang:1.13-alpine AS build-app
```

Comment:
I am thinking, we should try to keep our go version up to date as fast as possible. Could you bump to 1.14?

Reply:
Ok, no problem going to 1.14 I think